backend/settings.py

```diff
@@ -0,0 +1 @@
+limit = 10
```

Comment:
read it from an environment variable like `BEERAPP_PAGE_LIMIT`:
`limit = int(os.getenv('BEERAPP_PAGE_LIMIT',10))`

Constants should be capitalized: `PAGE_SIZE`